src/deploy/hosting/prepare.ts

```diff
@@ -9,7 +9,7 @@ import * as deploymentTool from "../../deploymentTool";
 /**
  *  Prepare creates versions for each Hosting site to be deployed.
  */
-export async function prepare(context: any, options: any): Promise<void> {
+export async function prepare(context: any, options: any, payload: any): Promise<void> {
```

Comment:
If possible, it would be great to give these real types while you're here. 

options has a generic type that can be used for almost any command (defined in options.ts).
 In other deploy flows, we've defined types for Context/Payload as well (see https://github.com/firebase/firebase-tools/blob/master/src/deploy/extensions/args.ts#L3 and https://github.com/firebase/firebase-tools/blob/master/src/deploy/functions/args.ts#L9)

This also has the added benefit of silencing the many, many lint warnings that pop up in this file during reviews.